workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+import classNames from 'classnames';
+
+import {
+  Page,
+  useChangelogPages,
+  useDocumentationPageLink,
+} from '<src>/components';
+import { useBatchCommits } from '<src>/hooks/useBatchCommits';
+import { useSpacingStyles, useUtilityStyles } from '<src>/styles';
+import { formatTimeAgo } from '<src>/utils';
+
+export const ChangelogHistory: FC = () => {
+  const containerSpacing = useSpacingStyles({
+    size: 3,
+  });
+  const batchCommitSpacing = useSpacingStyles({});
+  const utilityStyles = useUtilityStyles({});
+  const { loading, batchCommits } = useBatchCommits();
+  const changelogPage = useChangelogPages();
+  const documentationPage = useDocumentationPageLink();
+  const history = useHistory();
+
+  return (
+    <Page>
+      <Page.Navbar />
+      <Page.Body
+        padded
+        className={containerSpacing.padding}
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+        }}
+        loading={loading}
+      >
+        <h1>Changelog</h1>
+        <div style={{ width: 800, display: 'flex', flexDirection: 'column' }}>
```

Comment:
For things like layout, and width - I think inline styling actually works well here - usually you want them granular enough that you can be specific (e.g. display flex, with all the other required css styles)